README.md

```diff
@@ -75,6 +76,7 @@ Retries are disabled by default, but you can set maxTestRetryFailures and maxTes
 The following example declares -PmaxTestRetries=1 and -PmaxTestRetryFailures=3 to enable a failed test to be retried once, with a total retry limit of 3.
 
     ./gradlew test -PmaxTestRetries=1 -PmaxTestRetryFailures=3
+    ./gradlew test -Pkafka.test.run.flaky=true -PmaxTestRetries=1 -PmaxTestRetryFailures=3
```

Comment:
I don't think we need to add this command in this section, this section is focus on `Specifying test retries`

Reply:
@mingdaoy could you please address @m1a2st comment?